Libraries/LibCompress/Deflate.cpp

```diff
@@ -392,7 +390,8 @@ CanonicalCode::CanonicalCode(Vector<u8> codes)
     }
 }
 
-i32 binary_search(Vector<u32>& heystack, u32 needle)
+// FIXME: Use AK::Span::binary_search instead
```

Comment:
Span doesnt have a binary_search as far as i can see?
There is one in AK, but since it returns the element instead of the index, its not usable here

Reply:
Oh! I'll remove it then.